README.md

```diff
@@ -14,6 +14,13 @@ methods for ensuring high code quality:
 Tool-support has gotten pretty good, so all of these are rather quick to set
 up. Where possible checks should be automated and have shareable configs.
 
+## Automated Code Formatting
+
+[Prettier](https://github.com/prettier/prettier) is a tool to automatically
+format source code. It can be integrated into your Git workflow and editor.
+
+Note that [using Prettier does not make linting obsolete](https://github.com/prettier/prettier#how-does-it-compare-to-eslint-or-tslint-stylelint).
```

Comment:
maybe add a little more detail for the curious:


> Note that [using Prettier does not make linting obsolete](https://github.com/prettier/prettier#how-does-it-compare-to-eslint-or-tslint-stylelint), since prettier solves code _formatting_ problems but does not warn about code _quality_ problems as a linter does.